scripts/changelog/pr_utils.py

```diff
@@ -0,0 +1,15 @@
+import requests
+
+
+def fetch_pull_request(pr_number, token):
+    prs_url = "https://api.github.com/repos/mapbox/mapbox-navigation-android/pulls"
+    pr_url = prs_url + "/" + pr_number
+
+    headers = {"Authorization": "Bearer " + token}
+    return requests.get(pr_url, headers=headers).json()
+
+def is_draft(pr):
```

Comment:
I thought the `ready_for_review` made sure the action does not run for draft pull requests 

https://github.com/mapbox/mapbox-navigation-android/blob/8322e433344e41290eeab3a33f8e2282b91faee9/.github/workflows/rename_changelog_file.yml#L4

There is probably a good solution to also, conditionally run if the label exists 🤔 , maybe something like this? 

https://stackoverflow.com/questions/62325286/run-github-actions-when-pull-requests-have-a-specific-label

Reply:
> I thought the `ready_for_review` made sure the action does not run for draft pull requests.

Yeah, but we also need `synchronize` which will run every time PR is updated.

> There is probably a good solution to also, conditionally run the action based on the labels? 🤔

Do you mean when the "skip changelog" label has been removed?
It would probably be something like this.

```yaml
name: Rename changelog files
on:
  pull_request:
    types: [ opened, reopened, ready_for_review, synchronize, unlabeled ]
jobs:
  process:
    if: github.event.action != 'unlabeled' || (github.event.action == 'unlabeled' && github.event.label.name == 'skip changelog')
   ...
```

Reply:
Done. I've achieved the desired logic by using GitHub expressions only.